extensions/consensus/defaults.go

```diff
@@ -0,0 +1,78 @@
+package consensus
+
+import (
+	"context"
+	"encoding/hex"
+	"errors"
+	"math/big"
+
+	"github.com/kwilteam/kwil-db/common"
+	"github.com/kwilteam/kwil-db/common/chain"
+	"github.com/kwilteam/kwil-db/common/chain/forks"
+	"github.com/kwilteam/kwil-db/common/functions"
+	"github.com/kwilteam/kwil-db/core/types/transactions"
+)
+
+// Register the canonical (non-extension) hard forks that are baked into kwild.
+// I'd rather have canonical hard forks in common/chain, but then we have a
+// circular import trying to use RegisterHardfork from there.
+func init() {
```

Comment:
Is this supposed to have a build tag? I assume we don't actually want to include this in user apps.